doc/modules/model_evaluation.rst

```diff
@@ -74,9 +74,9 @@ Scoring                      Function                                    Comment
 'adjusted_rand_score'        :func:`metrics.adjusted_rand_score`
 
 **Regression**
-'mean_absolute_error'        :func:`metrics.mean_absolute_error`
-'mean_squared_error'         :func:`metrics.mean_squared_error`
-'median_absolute_error'      :func:`metrics.median_absolute_error`
+'neg_mean_absolute_error'    :func:`metrics.mean_absolute_error`         Negative MAE returned so greater=better
```

Comment:
I'm not sure if this explanation is the best possible but I don't have a better one ;)
\